docs/apache_spark.md

```diff
@@ -2,42 +2,33 @@
 template: full_width_with_menu.html
 ---
 
-# Apache Spark
+# How to use Apache Spark with the CERN SWAN service
```

Comment:
This title will be used as the text in the left menu, so this seems a bit too big.
Other option is to configure in mkdocs.yml an alias (I believe this is possible).